docs/spark-connector/spark-catalog-hive.md

```diff
@@ -0,0 +1,64 @@
+---
+title: "Spark connector hive catalog"
+slug: /spark-connector/spark-catalog-hive
+keyword: spark connector hive catalog
+license: "Copyright 2024 Datastrato Pvt Ltd.
+This software is licensed under the Apache License version 2."
+---
+
+With the Gravitino Spark connector, accessing data or managing metadata in Hive catalogs becomes straightforward, enabling seamless federation queries across different Hive catalogs.
+
+## Capabilities
+
+Supports most DDL and DML operations in SparkSQL, except such operations:
+
+    * Function operations 
+    * Partition operations
+    * View operations
+    * Querying UDF
+    * `LOAD` clause
+    * `CREATE TABLE LIKE` clause 
+
+## Requirement
+
+* Hive metastore 2.x
+* HDFS 2.x or 3.x
+
+## Table properties
+
+Supports both datasource format and hive format when creating tables.
+
+Create tables with hive format:
+
+```sql
+CREATE [ EXTERNAL ] TABLE [ IF NOT EXISTS ] table_identifier
+    [ ( col_name1[:] col_type1 [ COMMENT col_comment1 ], ... ) ]
+    [ COMMENT table_comment ]
+    [ PARTITIONED BY ( col_name2[:] col_type2 [ COMMENT col_comment2 ], ... ) 
+        | ( col_name1, col_name2, ... ) ]
+    [ CLUSTERED BY ( col_name1, col_name2, ...) 
+        [ SORTED BY ( col_name1 [ ASC | DESC ], col_name2 [ ASC | DESC ], ... ) ] 
+        INTO num_buckets BUCKETS ]
+    [ ROW FORMAT row_format ]
+    [ STORED AS file_format ]
+    [ LOCATION path ]
+    [ TBLPROPERTIES ( key1=val1, key2=val2, ... ) ]
+    [ AS select_statement ]
+```
+
+Create tables with datasource format:
+
+```sql
+CREATE TABLE [ IF NOT EXISTS ] table_identifier
+    [ ( col_name1 col_type1 [ COMMENT col_comment1 ], ... ) ]
+    USING data_source
+    [ OPTIONS ( key1=val1, key2=val2, ... ) ]
+    [ PARTITIONED BY ( col_name1, col_name2, ... ) ]
+    [ CLUSTERED BY ( col_name3, col_name4, ... ) 
+        [ SORTED BY ( col_name [ ASC | DESC ], ... ) ] 
+        INTO num_buckets BUCKETS ]
+    [ LOCATION path ]
+    [ COMMENT table_comment ]
+    [ TBLPROPERTIES ( key1=val1, key2=val2, ... ) ]
+    [ AS select_statement ]
+```
```

Comment:
Do you need to add more SQL examples about DDL, DML and DQL to show users how to use it.

Reply:
ok